tracer/src/Datadog.Trace/DuckTyping/DuckType.cs

```diff
@@ -904,7 +944,23 @@ private static void CreatePropertiesFromStruct(TypeBuilder? proxyTypeBuilder, Ty
                 switch (duckAttribute.Kind)
                 {
                     case DuckKind.Property:
-                        PropertyInfo? targetProperty = targetType.GetProperty(duckAttribute.Name, duckAttribute.BindingFlags);
+                        PropertyInfo? targetProperty = null;
+                        if (duckAttribute.Name.IndexOf(',') == -1)
+                        {
+                            targetProperty = targetType.GetProperty(duckAttribute.Name, duckAttribute.BindingFlags);
```

Comment:
Seeing as you've added duck typing features, we should probably add unit tests for it too 😉 

I assume we should cover:
- DuckField with multiple names
- DuckProperty with multiple names
- Reverse duck types with multiple names?

Reply:
+1

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3616/commits/faa23646d03e2a672a2cd3f22e3b25c0e2838ce3